DeviceComunication/DeviceComunication.md

```diff
@@ -469,9 +447,9 @@ If error numbers 1,3 and 5 occur 4 hours after restart, the device proceeds as f
 If the error number 1,3,4 and 5 occurs within 4 hours of restart, the device proceeds as follows:
 
 1. The LED flashes X times in 10 cycles (according to the [notifications table](#led-notifications)) to indicate an error.
-2. The device will check if 4 hours have passed since the restart.
-3. If the time has not passed, it is put to sleep for 2 minutes and then repeats points 1 and 2.
-4. If the time has passed, the device restarts and continues to function as if it were [restarted](#standard-restart).
+2. The device restarts and continues to operate as [usual](#standard-restart).
```

Comment:
Komentář k odstavci o kus výš - _If a fatal error occurs on the device, the device will reset and send an error message ..._

Upravit pořadí podle skutečnosti ...